bfdd/bfd_packet.c

```diff
@@ -893,10 +901,31 @@ int bp_set_tos(int sd, uint8_t value)
 	return 0;
 }
 
+static int bp_set_reuse_addr(int sd)
```

Comment:
Can't we just use this function as bool?

Reply:
yes